Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -38,6 +39,11 @@ export default function ApplicationViews({ isLoggedIn }) {
         <Route path="/Tags">
```

Comment:
could you discuss with your team what a good naming convention strategy for your routes will be. For example, /tags/add or something like that? also make sure they are lower-cased, just to match everyone else's.

Reply:
@joshdbarton Made it consistent with the naming conventions with teams' category route path by changing the path to a lower case tag and adding exact to the end of line 39. Also changed Line 47 to /tag/new to make everything match.